src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java

```diff
@@ -5,30 +5,55 @@
 import com.github.vkorobkov.jfixtures.config.yaml.Node;
 import com.github.vkorobkov.jfixtures.util.YmlUtil;
 import lombok.extern.slf4j.Slf4j;
+import lombok.val;
 
 import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.stream.Stream;
 
 @Slf4j
-public class ConfigLoader {
+public final class ConfigLoader {
     private static final String CONF_YML = ".conf.yml";
-    private final Path path;
+    private static final String CONF_YAML = ".conf.yaml";
 
-    public ConfigLoader(String fixturesRoot) {
-        this.path = Paths.get(fixturesRoot).resolve(CONF_YML);
+    private ConfigLoader() {
     }
 
-    public Root load() {
-        return new Root(loadRootNode());
+    public static Root load(String fixturesRoot) {
+        return new Root(loadRootNode(fixturesRoot));
     }
 
-    private Node loadRootNode() {
-        try {
-            return Node.root(YmlUtil.load(path));
-        } catch (IOException exception) {
-            log.warn("file '.cong.yml' not found, using defaults");
-            return Node.emptyRoot();
+    private static Node loadRootNode(String fixturesRoot) {
```

Comment:
Given [we have this](https://github.com/vkorobkov/jfixtures/pull/84/files#r146479303) the whole `loadRootNode` could be simplified significantly:
```java
try {
  val path = getConfigPath(fixturesRoot).get();
  return Node.root(YmlUtil.load(path));
}
catch(NoSuchElementException) {
  log.warn("Neither file '" + CONF_YAML + "' nor '" + CONF_YML + "' not found, using defaults");
}
catch(IOException e) {
  log.warn("Config loading failed, using the defaults. Details: " + e);
}
return Node.emptyRoot();
```

Reply:
Maybe even we need to `log.info` instead of `log.warn` when we catch `NoSuchElementException` since missing config file is a normal thing to have